Config/config.php

```diff
@@ -34,6 +34,28 @@
                 ],
             ],
         ],
+        'api'    => [
+            'mautic_api_contactsourcestandard'      => [
+                'standard_entity' => true,
+                'name'            => 'contactsources',
+                'path'            => '/sources',
+                'controller'      => 'MauticContactSourceBundle:Api\Api',
+            ],
+            'mautic_api_contactsource_add_campaign' => [
+                'path'       => '/sources/{contactSourceId}/campaign/add',
+                'controller' => 'MauticContactSourceBundle:Api\Api:addCampaign',
+                'method'     => 'PUT',
+            ],
+            'mautic_api_contactsource_new_campaign' => [
+                'path'       => '/campaigns/create',
```

Comment:
Why use this instead of https://developer.mautic.org/#create-campaign ?
GET action is not RESTful for creation.

Are we inadvertently giving the ability to create campaigns to sources/editors, because that's what this looks like. Otherwise I don't understand this direction. Are we checking permissions to ensure the user can create campaigns? You'll need another checkEntityAccess there likely.